website/static/js/pages/project-settings-page.js

```diff
@@ -40,7 +40,7 @@ initNotificationsTB();
 
 //Initialize treebeard grid for node admins
 var ProjectMailingList = require('../mailingListTreebeard.js');
-var mailingListSettingsURL = ctx.node.urls.api + 'mailing_list/';
+var mailingListSettingsURL = ctx.node.urls.api + 'mailing_list/';  // TODO [OSF-6400]: Update to V2
```

Comment:
This is where the viewmodel is instantiated. I can prevent the other one by passing this url in to the viewmodel.